sfge/src/main/java/com/salesforce/graph/symbols/apex/ApexForLoopValue.java

```diff
@@ -153,6 +153,10 @@ public Optional<ApexValue<?>> apply(MethodCallExpressionVertex vertex, SymbolPro
 
     @Override
     public Optional<String> getDefiningType() {
+        if (!items.isEmpty()) {
+            // TODO: we should try to look at more than just the first item
+            return items.get(0).getDefiningType();
+        }
```

Comment:
This change helps ApexForLoopValue have a defining type that helps this get a method match in `MethodUtil.java`.